src/ur3e_hande_moveit_config/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ur3e_hande_moveit_config</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="wfernando1@bnl.gov">wfernando1</maintainer>
+  <license>TODO: License declaration</license>
```

Comment:
Add this. It will be BSD-3 so long as that is compatible with what you've borrowed for this. (It is.)

Reply:
Description and license added: https://github.com/maffettone/erobs/blob/713573eeb9b4f04c3dcd55634bd55a72deedef19/src/ur3e_hande_moveit_config/package.xml#L6-L8